src/Model/CompositeTypeGo.cs

```diff
@@ -26,12 +26,16 @@ public class CompositeTypeGo : CompositeType
 
         private CompositeTypeGo _rootType;
 
-        public CompositeTypeGo()
+        public CompositeTypeGo() : this("")

```

Comment:
Does this have any unintended side-effects?  Probably not but would be good to regenerate the entire SDK as part of validating this PR (perhaps you already did that).

Reply:
yes I regenerated all is good